src/main/java/com/thebois/models/world/terrains/Sand.java

```diff
@@ -0,0 +1,44 @@
+package com.thebois.models.world.terrains;
+
+import com.thebois.models.Position;
+
+/**
+ * Terrain of type sand.
+ */
+public class Sand extends AbstractTerrain {
+
+    /**
+     * Instantiates a new sand terrain tile.
+     *
+     * @param x The X position of the tile.
+     * @param y The Y position of the tile.
+     */
+    public Sand(final float x, final float y) {
+        super(x, y);
+    }
+
+    /**
+     * Instantiates a new sand terrain tile.
+     *
+     * @param position The position of the terrain tile.
+     */
+    public Sand(final Position position) {
+        this(position.getPosX(), position.getPosY());
```

Comment:
style

All the other terrain subtypes do it the other way around, where the other constructor calls this one. Should probably be consistent.